taca/backup/backup.py

```diff
@@ -179,10 +196,12 @@ def _get_run_type(self, run):
                 "^(\d{8})_(\d{4})_([1-3][A-H])_([0-9a-zA-Z]+)_([0-9a-zA-Z]+)$", run
             ):
                 run_type = "promethion"
+            elif "_AV" in run:
```

Comment:
Is there a reason we can't use `filesystem.RUN_RE_ELEMENT` for this too? I see that the RUN_RE_ONT is slightly different

Reply:
Good point. For ONT the difference is to distinguish between minion and promethion, `RUN_RE_ONT` catches both of them. But for Aviti we could use `RUN_RE_ELEMENT`.